locale/reference/reference_lua/box_error.pot

```diff
@@ -260,3 +266,58 @@ msgstr ""
 msgid "The iProto protocol also supports stacked diagnostics. See details in :ref:`Binary protocol -- responces for errors -- extra <box_protocol-responses_error_extra>`."
 msgstr ""
 
+msgid "Custom error types"
+msgstr ""
+
+msgid "Errors can be created in 2 ways: with ``box.error.new()`` and with ``box.error()``."
+msgstr ""
+
+msgid "Both used to take either code, reason, <reason string args> or {code = code, reason = reason, ...} arguments."
+msgstr ""
+
+msgid "In the first option instead of code you can specify a string as its own error type. In the second option you can specify both code and type."
+msgstr ""
+
+msgid "box.error('MyErrorType', 'Message')\n"
+"box.error({type = 'MyErrorType', code = 1024, reason = 'Message'})"
+msgstr ""
+
+msgid "Or no-throw version:"
+msgstr ""
+
+msgid "box.error.new('MyErrorType', 'Message')\n"
+"box.error.new({type = 'MyErrorType', code = 1024, reason = 'Message'})"
+msgstr ""
+
+msgid "When a custom type is specified, it is shown in ``err.type`` attribute. When it is not specified, ``err.type`` shows one of built-in errors such as ``'ClientError'``, ``'OurOfMemory'``, etc."
+msgstr ""
+
+msgid "Name length limit on the custom type is *63 bytes*. Everything longer then the limit is truncated."
+msgstr ""
+
+msgid "Original error type can be checked using ``err.base_type`` member, although normally it should not be used. For user-defined types base type is ``'CustomError'``."
```

Comment:
please update the pot file

Reply:
updated in ff7edbb556e6722f82d14374372a827abeb0912a